lib/Control/Control.js

```diff
@@ -1663,7 +1663,7 @@ Control.concat = function (ctor, props, instance) {
 
 	if (props.style) {
 		if (instance) {
-			str = (proto.style ? (proto.style + ';') : '') + (props.style + ';');
+			str = (proto.style ? proto.style : '') + props.style;
```

Comment:
Will this open us up to the possibility of style concatenation without a semicolon between the style properties of the parent kind and the child kind (i.e. if the developer leaves off the trailing semicolon in the parent kind's `style` property)? The appearance of double semicolons was not an issue before because we would run the concatenated style string through the `normalizeCssStyleString` method: https://github.com/enyojs/enyo/blob/2.4.0/source/dom/Control.js#L1106.